### ISSUE TITLE
vaadin-confirm-dialog 1.2.0 breaks the npm dependencies

### ISSUE THREAD

**Issue by alump** [vaadin/vaadin-rich-text-editor-flow#99]
When using Vaadin 14.1.x series. If I add vaadin-rich-text-editor-flow dependency... it will break the client side once RichTextEditor gets included to bundle.

This happens because @vaadin/vaadin-rich-text-editor has @vaadin/vaadin-confirm-dialog dependency ^1.1.0 -> which loads 1.2.0. Which then ends up depending on @vaadin/vaadin-lumo-styles ^1.6.0.

Same time @vaadin/flow-deps has dependency to 1.5.0 of lumo-styles.

This causes duplicate dependencies that breaks the client side.

**Comment by alump** [comment]
Workaround, have maven dependency to vaadin-confirm-dialog-flow in you project that will force it to stay in 1.1.x versions assuming you use the managed dependency version 2.0.4.

But I assume the NPM should be correct? Or then vaadin-rich-text-editor-flow should bring in maven dependency to force version to be right... if NPM version issues won't be resolved.

**Comment by alvarezguille** [comment]
Just to verify, are you using `vaadin-core` and then adding the pro dependencies one by one?

**Comment by alump** [comment]
Yes. only free vaadin dependencies (aka core), then add vaadin-confirm-dialog-flow dependency and issue above happens.

**Comment by alump** [comment]
project has com.vaadin vaadin-bom dependency of 14.1.4

**Comment by alump** [comment]
Tried to clean up everything, and rebuild things. Now I didn't get 1.6.0 version of vaadin-lumo-styles. So I don't know what caused it pull 1.6.0 when I added vaadin-rich-text-editor dependency. Maybe it was some strange existing package-lock issue again that happened was it wasn't removed.

**Comment by fschon** [comment]
I have the same problem, I just don't know where it breaks the client side. 

When including dependencies:
```
        <dependency>
            <groupId>com.vaadin</groupId>
            <artifactId>vaadin-core</artifactId>
        </dependency>
        <dependency>
            <groupId>com.vaadin</groupId>
            <artifactId>vaadin-rich-text-editor-flow</artifactId>
        </dependency>
```
Then none of my Polymer JsModule templates will render on the clients. Other non polymer components render fine.

If I change <artifactId>vaadin-core</artifactId> to include all pro components <artifactId>vaadin</artifactId> then the Polymer templates display again.

I can't see any errors in the Javascript or server side logs.

**Comment by TatuLund** [comment]
The same issue can happen also with Crud.

**Comment by TatuLund** [comment]
See also https://github.com/vaadin/flow/issues/8475

**Comment by alvarezguille** [comment]
Should be fixed in 14.3 with pnpm, please reopen if it can be reproduced
